include/client/networkclient.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2017 CodiLime
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#pragma once
+
+#include <set>
+#include <map>
+#include <random>
+#include <memory>
+#include <cstdint>
+#include <iostream>
+#include <unordered_map>
+#include <unordered_set>
+
+#include <QString>
+#include <QTcpSocket>
+
+#include "network/msgpackwrapper.h"
+#include "data/nodeid.h"
+
+namespace veles {
+namespace client {
+
+class NodeTree;
+
+typedef std::pair<bool, std::shared_ptr<std::string>> pair_str;
```

Comment:
pair_str is actually being used in NetworkClient::sendMsgConnect. pair_uint64_t was useless and I have removed it.